samples/nats_jetstream/v3/receiver/main.go

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright 2024 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/nats-io/nats.go"
+	"github.com/nats-io/nats.go/jetstream"
+
+	cejsm "github.com/cloudevents/sdk-go/protocol/nats_jetstream/v3"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+func main() {
+	ctx := context.Background()
+
+	natsURL := "nats://localhost:4222"
+	natsSubject := "sample"
+	natsStream := "stream"
+
+	createStream(natsURL, natsStream, natsSubject)
+	consumerOpt := cejsm.WithConsumerConfig(&jetstream.ConsumerConfig{FilterSubjects: []string{natsSubject}})
+	urlOpt := cejsm.WithURL(natsURL)
+	protocol, err := cejsm.New(ctx, consumerOpt, urlOpt)
+	if err != nil {
+		log.Fatalf("failed to create nats protocol, %s", err.Error())
```

Comment:
nit: typically use `:` here (applies to others as well)

```suggestion
		log.Fatalf("failed to create nats protocol: %s", err.Error())
```